src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -207,6 +207,12 @@ export const ArtworkApp: React.FC<Props> = props => {
     <>
       <UseRecordArtworkView />
 
+      {!artwork.published && (
+        <FullBleedBanner variant="error">
+          <Text>This work is not published on Artsy yet.</Text>
```

Comment:
**suggestion:** Remove the "yet", which implies it's going to be published. Could just as easily be a work that's been unpublished.

```suggestion
          <Text>This work is not currently published on Artsy.</Text>
```

Reply:
Definitely! Believe it or not this was copilot generated text.